README.md

```diff
@@ -31,3 +32,28 @@ Active voice is preferred in most writing examples. Ex., "this ClusterPolicy mut
 * Kubernetes resource kinds are considered proper nouns and are distinguished from other nouns by the initial letter capitalization. Ex., "a Kubernetes Pod will be annotated".
 * Anything intended to be proper code or typed at a CLI is formatting using Markdown code syntax with backticks or in blocks (surrounded by three backticks).
 * Code represented in blocks should prefer a syntax declaration for this theme's highlighting ability. Ex., when displaying YAML notate the code block with three backticks and "yaml".
+
+## Managing Release Versions
+
+Here are the rules for managing release versions:
+
+1. All fixes and feature changes go to the `main` (we may in a few rare cases make fixes to prior versions of the documentation.) The main branch can be accessed at `https://main.kyverno.io`.
+
+2. When a new release is ready for GA, a new release branch is created (see steps below). Release branches are named `release-{major}-{minor}-{patch}` for example `release-1-4-2`. The release branch can be accessed using the `{branch}.nirmata.io` and the latest release is available at `kyverno.io`.
```

Comment:
It'd be `{branch}.kyverno.io` right? :)

Reply:
No, the latest release (e.g. currently 1.4.1) should be on https://kyverno.io as we update the Netlify setting.

Reply:
I know, but you put `nirmata.io` and not `kyverno.io`.

Reply:
Doh! Will fix.